contents/handbook/small-teams/growth/index.mdx

```diff
@@ -15,10 +15,19 @@ Maximize the number of people who get value out of PostHog, and help them realiz
 
 ## What the Growth team does
 
-- Sign up
-- Activation and reducing time to value
-- Billing and pricing changes
-- Product led growth, including messaging, emailing and notifications
+- Anything activation
+  - Activation to features or across features, but only for existing features.
+  - Onboarding flows (global and per feature)
```

Comment:
I thought we decided that we are responsible for: 
- global onboarding (eg getting someone onto the platform and taking any steps that are necessary before they can actually use any of the products)
- setting up systems and tooling for product-specific onboarding that can be leveraged for each product
- assisting product teams with getting the onboarding system set up initially

But that we are specifically _not_ responsible for activating into each individual product, because the product teams will have the most context there and will know what users need to learn most in order to see the most value. Ya? Or am I remembering that wrong?

Reply:
My understanding is that for now, we are responsible for helping teams define the base tooling for enabling a good activation flow (i.e. a new onboarding for each product). After we deliver on that, they can keep on iterating, but I think the cross-team tooling would still be on us.